### ISSUE TITLE
EYRA benchmark platform MVP

### ISSUE THREAD

**Issue by rzinkstok** [NLeSC/TEAM2018#47]
The Nijmegen Diagnostic Image Analysis Group has created the grand-challenge.org website for hosting benchmark challenges in the medical imaging domain. We want to extend this platform to become a multi/cross-domain benchmark challenge platform. In a previous sprint, a demo version of the grand-challenge.org site was deployed on a HPC cloud server. During this sprint, we will update this instance to the latest version and start extending it to suit our needs. We will implement a full REST API, add community-related functionality, and start working on a new React-based user interface. If time permits, we will implement a demo challenge as well.

**Comment by romulogoncalves** [comment]
@rzinkstok is there project hours for this work? If yes, which one?

**Comment by rzinkstok** [comment]
It's all part of EYRA, you need the project number/code for it?

**Comment by romulogoncalves** [comment]
Name is enough.

**Comment by romulogoncalves** [comment]
@rzinkstok were all the goals achieved? Could you summarize in 3 lines what was achieved? Can we consider it done and close this issue?

**Comment by rzinkstok** [comment]
- Set up a development pipeline, working on separate branches in the comic grandchallenges
- Set up PyCharm combining docker-compose support with the PyCharm django dev server (for debugging)
- Built a skeleton frontend using React
- Implemented basic API endpoints including swagger interface
